packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md

```diff
@@ -0,0 +1,162 @@
+Use the Devices SDK to add custom push verification functionality to your Android app.
+
+See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/android/main/) for general information about mobile authentication at Okta.
+
+> **Note:** The following sample code assumes that suspend functions are called in a coroutine scope. See [Kotlin Coroutines](https://kotlinlang.org/docs/coroutines-overview.html) for more information.
+
+<!-- Data flow DIAGRAM -->
+
+### Install the dependency
+
+Add the Okta Authenticator SDK dependency to your `build.gradle` file:
```

Comment:
readme also updated. This is now called Okta Devices SDK.

Reply:
Thank you. Good catch.